src/pylhe/__init__.py

```diff
@@ -222,38 +225,30 @@ def readNumEvents(file):
 
 
 def visualize(event, outputname):
+    # retrieve mapping of PDG ID to particle name as LaTeX string
```

Comment:
It would be great if this could be an actual docstring instead of just a comment.

Reply:
But this comment does not explain what the function is doing?

Reply:
Comments and docstrings are not actually the same thing. Check out [PEP 257](https://www.python.org/dev/peps/pep-0257/) for some of the differences, but the main thing that matters here is that if you ran

```python
>>> import pylhe
>>> help(pylhe.visualize)
```
you would just get
```
visualize(event, outputname)
```

and no information as the comment gets stripped out. Docstrings don't and so are actually useful documentation.

Reply:
Right, what is mean is that a proper docstring would describe the function as a whole, whereas the comment you're refering to just explains the one statement that follows it.

Reply:
(I also added a docstring.)